lib/phonemiser.py

```diff
@@ -1,7 +1,9 @@
-# Functions for doing grapheme to phoneme for supported languages
-# The default functions first do LRLM segmentation of the orthography
-# and then lookup the segments in a lookup table. Any symbols which
-# do not appear are discarded.
+"""Functions for doing grapheme to phoneme for supported languages The default.
```

Comment:
Looks like "The default moved lines and should be with the paragraph below instead.

Reply:
done.